src/components/nudge/Nudge.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import PrimaryButton from '../primary-button';
+import X16 from '../../icon/fill/X16';
+
+import './Nudge.scss';
+
+export interface NudgeProps {
```

Comment:
open thought: should we just do `children` props versus having specific ones for elements on the Nudge.

Reply:
Kind of depends on how flexible or rigid you want the structure to be.